spec/models/member_spec.rb

```diff
@@ -164,4 +164,36 @@
       end
     end
   end
+
+  describe "#other_income_types" do
```

Comment:
to test this, we can use a cool library called `shoulda` (which we are already using) https://github.com/thoughtbot/shoulda-matchers/blob/master/lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb

so we could replace all of these tests with:

```
it { should validate_inclusion_of(:other_income_tpes).in_array(Member::OTHER_INCOME_TYPES) }
it { should allow_value(nil).for(:other_income_types) }
```